submissions/kirill8210/dom-task/js/index.js

```diff
@@ -0,0 +1,65 @@
+import clubs from './json.js';
+import removeMenu from './mobile.js';
+
+const appClub = document.getElementById('main');
+const appUl = document.getElementById('menu');
+
+const createCardClub = (clubs) =>{
+    const {
+        name,
+        img,
+        description
+    } = clubs;
+
+    const clubCards = document.createElement('div');
+    clubCards.className = 'club';
+
+    clubCards.insertAdjacentHTML('afterbegin', `
+        <h2 class="club_title">${name}</h2>
+        <div class="club_about">
+            <div class="club_description">
+                ${description}
+            </div>
+            <div class="club_img"><img src="${img}" alt="${name}"></div>
+        </div>
+    `);
+
+    return clubCards;
+};
+
+const createUl = (clubs) =>{
+    const { name } = clubs;
+    const clubUl = document.createElement('li');
+
+    clubUl.insertAdjacentHTML('afterbegin', `
+        <a href="#" class="menu_el">${name}</a>
+    `);
+
+    return clubUl;
```

Comment:
```suggestion
    return clubUl.insertAdjacentHTML('afterbegin', `
        <a href="#" class="menu_el">${name}</a>
    `);
```

Reply:
Why cut? Your recommendation to shorten the code will call undefined.